pkg/network/setup/netpod/netpod.go

```diff
@@ -408,15 +408,10 @@ func (n NetPod) bridgeBindingSpec(podIfaceName string, vmiIfaceIndex int, ifaceS
 }
 
 func (n NetPod) networkQueues(vmiIfaceIndex int) int {
-	ifaceModel := n.vmiSpecIfaces[vmiIfaceIndex].Model
-	if ifaceModel == "" {
-		ifaceModel = v1.VirtIO
+	if n.vmiSpecIfaces[vmiIfaceIndex].Model == v1.VirtIO || n.vmiSpecIfaces[vmiIfaceIndex].Model == "" {
```

Comment:
nit: The common case is that the model is not specified, so I would swap the order.